sprint0/sprint0/Weapons/Arrow.cs

```diff
@@ -56,8 +58,9 @@ private void Move()
 
         public void Draw(SpriteBatch spriteBatch)
         {
+            LocationDraw = new Rectangle((int)Location.X, (int)Location.Y, (int)(width * Game1.Scale), (int)(height * Game1.Scale));
```

Comment:
this looks like the same rectangle as Location

Reply:
it is for when it is facing north or south, but not east or west